### PR TITLE
Improvements to runTest.sh

### PR DESCRIPTION
**Pull request by Vanderscycle**
using modern unix commands, I have improved the ux to run tests.

### CODE REVIEW

**Review by nodlesh — COMMENTED** [review]
What is the bifold/availableTests.txt used for?

**Comment by Vanderscycle** [comment]
Used by fzf to list all available tests to choose from.

Essentially used in so this command 
```
Bash ./runTest.sh -f bifold
```

**Comment by nodlesh** [comment]
I don't think we need to maintain another file like that. The tags and tests are already in the feature file. 
There is a bug with the manage script, but you should be able to do `./manage tests` and ./manage tags` to tell you what to run. 
The greps need to be updated in the manage script to do recursive like this. 
```
    ;;
  tags)
      grep -rh --include \*.feature @ aries-mobile-tests/features/* |  tr " " "\n" | sort -u | fmt
    ;;
.
.
.
  tests|scenarios)
      grep -rh --include \*.feature -B 1 Scenario aries-mobile-tests/features/*
    ;;
```
I've got an outstanding PR that when merged I'll fix this grep. I would suggest to use that. 
Let's also not get too wrapped up in this, things are going to change on the manage script side as we move forward and determine needs and wants. If this runTest helps you, then good, but don't make it too complex or add overhead.

**Comment by nodlesh** [comment]
I've just merged a PR that fixes the `./manage tests` and `./manage tags`.

File: runTest.sh [hunk 1]
```diff
@@ -1,34 +1,92 @@
...
+    rm "$1"
+  fi
+  touch "$1"
+}
+
+function fzf() {
+
+  logClear .runTest
+  bash ./manage tests | rg -e '(@[^\s]+)' | uniq |  command fzf --border -m | tee .runTest
+  bat .runTest
```
Comment (Vanderscycle) [inline runTest.sh]: @nodlesh I don't know how to parse the file to get only the uniques so that I may pass it to runTest()

**Comment by Vanderscycle** [comment]
I refactored the file and improved the workflow clarity on a different while I was trying to parse the input on ./manage tests

**Comment by ianco** [comment]
FYI you should rebase to reduce the number of commits in the PR

**Comment by Vanderscycle** [comment]
Oops, I'll revert that one

**Review by nodlesh — CHANGES_REQUESTED** [review]
Please remove the availableTests.txt. There will be a lot of tags in the feature files. We don't need another file to keep updated. If you want to pull tags get it from the `./manage tags` command.  Thanks.

**Pull request closed without merging**